app/src/main/java/org/projectbuendia/client/utils/Utils.java

```diff
@@ -408,6 +408,17 @@ public static DateTime getDateTime(Bundle bundle, String key) {
         return dateTime == null ? null : MEDIUM_DATETIME_FORMATTER.print(dateTime);
     }
 
+    /**
+     * @return True if birthdate is non-null and less than or equal to five years old
+     */
+    public static boolean isChild(LocalDate birthdate) {
+        if (birthdate != null){
+            Period age = new Period(birthdate, LocalDate.now());
+            return age.getYears() <= 5;
```

Comment:
This will return true for an age like 5 years + 11 months.  Because the fix is so tiny (`<` instead of `<=`) I'll just merge this and then make the fix.